app/src/main/java/de/tum/in/tumcampusapp/activities/PersonsSearchActivity.java

```diff
@@ -130,8 +129,21 @@ public void onStartSearch(String query) {
      */
     @Override
     public void onLoadFinished(PersonList response) {
-        if (response.getPersons() == null) {
```

Comment:
You still need to catch null values. Otherwise it might through a NullPointerException

Reply:
Ok, but then I don't get the "no search result" prompt if there is no result.

Reply:
You can do both. `if(null or isEmpty()`